frontends/analytics/package.json

```diff
@@ -29,7 +29,8 @@
     "tailwind-gradient-mask-image": "^1.2.0",
     "tailwind-scrollbar": "^3.1.0",
     "trieve-fetch-client": "*",
-    "yarn": "^1.22.22"
+    "yarn": "^1.22.22",
+    "zod": "^3.23.8"
```

Comment:
```suggestion
```
This doesn't seem to be used, can this be removed?

Reply:
Its used in SIngleQueryInfo to parse the v1 search metadata shape. Probably unnecessary for this use case but its a small dep and will very likely come in handy later based on the direction analytics is going.